ldi-core/version-object-creator/src/test/java/be/vlaanderen/informatievlaanderen/ldes/ldi/VersionObjectCreatorTest.java

```diff
@@ -158,20 +153,68 @@ void shouldMatchCountOfObjects(String fileName, String expectedId, LocalDateTime
 
 	@Test
 	void when_memberInfoExtractionFails_warningMessageIsLogged() {
-		Logger vocLogger = (Logger) LoggerFactory.getLogger(VersionObjectCreator.class);
-		ListAppender<ILoggingEvent> listAppender = new ListAppender<>();
-		listAppender.start();
-		vocLogger.addAppender(listAppender);
+		ListAppender<ILoggingEvent> listAppender = createListAppender();
+		Model inputModel = RDFParser.fromString("""
+				@prefix ex: <http://example.org/> .
+				@prefix time: <http://www.w3.org/2006/time#> .
+				@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+				
+				ex:John ex:hasAge "30" .
+				ex:John ex:hasName "John Smith" .
+				ex:John ex:hasFriend ex:Jane .
+				ex:John ex:created [
+				    a time:Instant ;
+				    time:inXSDDateTimeStamp "2023-08-18T13:08:00+01:00"^^xsd:DateTime
+				  ]
```

Comment:
This input turtle looks more like N-Triples (or N-Quads), although it is valid turtle. However: this is more turtle like representation:
```
@prefix ex: <http://example.org/> .
@prefix time: <http://www.w3.org/2006/time#> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .

ex:John
	ex:hasAge "30" ;
	ex:hasName "John Smith" ;
	ex:hasFriend ex:Jane ;
	ex:created [
    	    a time:Instant ;
    	    time:inXSDDateTimeStamp "2023-08-18T13:08:00+01:00"^^xsd:DateTime
  	] .
```